utilities/email_helper.py

```
@@ -0,0 +1,322 @@
+    """
+    try:
+        await db.execute(
+            update(League)
+            .where(League.id == league_id)
+            .values(enable_email_recaps=enable)
+        )
+        await db.commit()
+        
+        status = "enabled" if enable else "disabled"
+        print(f"Email recaps {status} for league {league_id}")
+        return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)